test/GoodGhosting_Attacker_Sends_DAI_Externally.test.js

```diff
@@ -155,10 +156,11 @@ contract("GoodGhosting", (accounts) => {
             // starts from 1, since player1 (loser), requested an early withdraw
             for (let i = 1; i < players.length; i++) {
                 const player = players[i];
+                const userDaiBalance = new BN(await token.methods.balanceOf(player).call({ from: admin }));
                 const result = await goodGhosting.withdraw({ from: player });
                 truffleAssert.eventEmitted(result, "Withdrawal", (ev) => {
                     console.log(`player${i} withdraw amount: ${ev.amount.toString()}`);
-                    return ev.player === player;
+                    return ev.player === player && !new BN(ev.amount.toString()).eq(userDaiBalance);
```

Comment:
Let me know your thoughts about this:
personally, I'd like to perform the calculations here on the test to check for the exact match that users would receive, since we know the numbers.

If you feel like that is too much, then we should at least compare that the event amount is greater than the previous user balance (prior to withdraw), instead of just different (it should never be less, so we should make sure that doesn't happen).

Reply:
I used gt actually earlier but couldn't find the right function

Reply:
Weird. I'd expect `gt` to work based on the docs (https://github.com/indutny/bn.js/#utilities).

Reply:
Let me debug again